ipynb/observationTable.md

```diff
@@ -0,0 +1,164 @@
+
```

Comment:
@sjanssen2 are the contents of this md and the *.py file the same? If so just maintain the md and you can get rid of the *.py